Helper.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+public class Helper
+{
+    protected List<char> _listOfLettersToCount;
+    protected List<char> _listOfNumberToCount;
+    private List<char> _illegalalCharacters;
+    private int? _result;
+
+    public Helper()
+    {
+        _listOfLettersToCount = ['a', 'b', 'c'];
+        _listOfNumberToCount = ['1', '2', '3'];
+        _illegalalCharacters = ['-', '<', '>'];
+    }
+
+    public bool IsCharacterAndIfINeedToCount(char character)
+    {
+        if (char.IsAsciiLetter(character)) 
+        {
+            var letter = character;
```

Comment:
you could simply use the 'character' that has been passed, no need for a new variable